source/store.ts

```diff
@@ -0,0 +1,190 @@
+// /source/store.ts
```

Comment:
I kind of preferred the old `memcached-store.ts` filename.

(Relatedly, I'd prefer if we renamed `lib.ts` to `express-rate-limit.ts` in that repo, but I didn't notice the name when I first reviewed it and never bothered to say anything afterwards.)